.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Python application
```

Comment:
This is a golang function and small libraries collection.

Reply:
please ignore this PR, I'm doing some security testing